pkg/apis/ceph.rook.io/v1/types.go

```diff
@@ -339,3 +341,15 @@ type GaneshaServerSpec struct {
 	// Resources set resource requests and limits
 	Resources v1.ResourceRequirements `json:"resources,omitempty"`
 }
+
+func (cs *ClusterStatus) SetState(to ClusterState) {
```

Comment:
Since `SetState`, `SetMessage`, and `SetCephName` simply assign the property, could remove them and just expect the caller to assign to the properties?